tutorials/securing-gcs-static-website/index.md

```diff
@@ -0,0 +1,455 @@
+---
+title: Securing a static website hosted on Google Cloud Storage 
+description: This tutorial describes a solution to protect static assets on Cloud Storage based on the GCLB Serverless Network Endpoint Groups (NEGs).
+author: shenxiang
+tags: Serverless, Cloud Storage, GCLB, Network Endpoint Groups
+date_published: 2020-08-26
+---
+
+It's easy to create and maintain an[ HTTPS-based static website on Google Cloud Storage](https://cloud.google.com/storage/docs/hosting-static-website)(GCS) with Cloud CDN, Google Cloud Load Balancer(GCLB), managed SSL certificates, and custom domains. This serverless approach becomes popular due to the flexibility, scalability, and low cost.   
+However, it's still challenging to provide authentication and authorization for a static website on GCP **in a serverless fashion**.  This tutorial describes a solution to protect static assets on GCS based on the GCLB [Serverless Network Endpoint Groups](https://cloud.google.com/load-balancing/docs/negs/setting-up-serverless-negs) (NEGs).  
+Below is the high-level architecture of this solution:
+
+![gcs-static-arch](static-website/src/assets/arch-gcs-site.png)
+
+The architecture above incorporates the following key features:
+
+1. Create a load balancer to use a custom domain with a managed SSL certificate. When a request is received, we use [routing rules](https://cloud.google.com/load-balancing/docs/https/setting-up-query-and-header-routing#http-header-based-routing) to check whether [a signed cookie for Cloud CDN](https://cloud.google.com/cdn/docs/private-content#signed_cookies) is in the request header. If the cookie doesn't exist, redirect the request to a login app hosted on Cloud Run. If there is a signed cookie, send the request to the Cloud CDN of the backend bucket.
```

Comment:
Keep these as nouns - remove the "create" verb ... "A load balancer using a custom domain..."

Reply:
Updated as suggested.